client/src/components/InviteGuests.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+
+
+
+const InviteGuests = ({ sessionId }) => {
+  return (
+    <div>
+      <h2>Invite your friends</h2>
+        {sessionId}
```

Comment:
For future, put this into a span so it can be styled easier. I'll do this in the next CSS pr.